content/blog/2025-03-27-valkey-8-1-0-rc1.md

```diff
@@ -0,0 +1,282 @@
+
+Replication traffic efficiency was also improved in 8.1 by offloading the reading of replication stream on the replicas to the I/O thread pool which means they can serve more read traffic. On the primaries, replication stream writes are now offloaded to the I/O thread pool.
+
+### Replication Improvements
+
+Full syncs with TLS enabled benefit from an [18%](https://github.com/valkey-io/valkey/pull/1479 ) improvement by removing CRC checksummming.
+
+The fork copy-on-write memory overhead is reduced by up to [47%](https://github.com/valkey-io/valkey/pull/905)
+ 
+### Sorted set and hyperloglog and bitcount optimisations
+
+`ZRANK` command, which serves a popular usecase in operating Leaderboards, was optimised to perform up to [45%](https://github.com/valkey-io/valkey/pull/1389) faster, depending on the serted set size.
+
+The probabilistic hyperloglog is another great data type, used for counting unique elements in very large datasets whilst using only 12KB of memory regardless of the amount of elements. By using the modern CPUs Advanced Vector Extensions of x86, Valkey 8.1 can achieve a [12x](https://github.com/valkey-io/valkey/pull/1293) speed for the operations like `PFMERGE` and `PFCOUNT` on hyperloglog data types.
+
+Similarly, the BITCOUNT operation has been improved up to [514%](https://github.com/valkey-io/valkey/pull/1741) using AVX2 on x86.
+
+### Other Performance Enhancements
+
+Active Defrag has been improved to eliminate large latencies to below [1ms](https://github.com/valkey-io/valkey/pull/1242) and anti-starvation protection has been introduced in the presence of long-running commands.
+
+Periodic processing/monitoring of clients has been moved off of the 100ms `serverCron()` interval loop, reducing system [overhead](https://github.com/valkey-io/valkey/pull/1387)
+ when large numbers of clients are present. 
+
+## Observability
+
+There are also several improvements to the observability of the system behavior in Valkey 8.1.
+
+### Log Improvements
+
+Valkey 8.1 brings new options to the format of the log file entries as well as the way timestamps are recorded in the log file. This makes it easier to consume the log files by log collecting systems. 
+
+The format of the log file entries is controlled by the `log-format` parameter, where the default is the existing format :
+
+- `legacy`: the default, traditional log format
+- `logfmt`: a structured log format; see https://www.brandur.org/logfmt
+
+The formatting of the timestamp of the log file entries is controlled by the `log-timestamp-format` parameter, where the default is the existing format: 
+
+- `legacy`: default format
+- `iso8601`: ISO 8601 extended date and time with time zone, of the form yyyy-mm-ddThh:mm:ss.sss±hh:mm
+- `milliseconds`: milliseconds since the epoch
+
+An example of the format `logfmt` with the timestamp format `iso8601`:
```

Comment:
I like everything til the examples - I would suggest omitting these examples (either to their own blog or links to the PR/issues).